### PR TITLE
171648108-Setup redux:(setting up redux and store)

### PR DESCRIPTION
**Pull request by brianineza01**
#### Type of change

Please select the relevant option

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Others (cosmetics, styling, improvements)
- [ ] This change requires a documentation update

#### How Has This Been Tested?

- [ ] Unit
- [ ] Integration
- [ ] End-to-end
- [x] No testing required

#### How to Test
N/A
#### Any background context you want to add (Optional)

N/A
#### STORY ID

[Finishes #171648108]

### CODE REVIEW

File: src/Components/App.jsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-				<Route exact path="/signin" component={SignIn} />
-			</Switch>
-		</div>
-	</Router>
+  <Provider store={store}>
+    <Router>
+      <div>
+        <NavBar />
+        <Switch>
+          <Route exact path='/' component={Home} />
```
Comment (placideirandora) [inline src/Components/App.jsx]: Put back the Double Quotes.  They are causing ESLint issues. These paths need to be enclosed in double-quotes as they were before.

**Review by placideirandora — CHANGES_REQUESTED** [review]
Open each and every file and fix those ESLint issues. ESLint is complaining a lot and once I put the file to check, If I find the issues.

File: src/Components/App.jsx (hunk 1 above)
Comment (brianineza01) [reply src/Components/App.jsx]: Done

Comment (brianineza01) [reply src/Components/App.jsx]: Feedback implemented

**Review by placideirandora — APPROVED** [review]

**Pull request merged**